### DISCUSSION TITLE
Introduce custom setting for how far ride music can travel when ride is out of focus

### DISCUSSION THREAD

**Discussion by Xtreme97-RCT** [OpenRCT2/OpenRCT2#20315] [Feature requests]
One problem I have sometimes when viewing parks is that when you have multiple rides within a moderate distance of eachother that all have custom music enabled, the music is able to travel quite far and can often overlap to make it too noisy for the user or ruin the atmosphere/presentation of a particular area. For example, being able to hear Pirate themed music in an Egyptian themed area would sound out of place when viewing the park.

A suggestion to solve this would be to have a custom setting either in the Options menu (so it would be game-wide) or on a per-ride basis (ie. in the Music tab of a ride) that would allow a user to adjust how far the music can travel when the ride is not in focus. This would give users more control in where ride music can be heard within a park.

**Comment by Richard-L** [comment]
Had this been changed at all from vanilla?

I feel due to wide-screen and native resolution support where more fits in the viewport, this has become an issue. But I could be mistaken.

**Comment by Liampie** [comment]
I suggested something similar a few years ago, but with different reasoning.

_"[...] the game was made for [...] 1280x1024 at most. Now with OpenRCT there probably is a lot more going on on the screen, and it results in too many sounds being played at the same time. If you play at a 1600x900 resolution, your screen can encompass an entire small park and you'll hear every single thing that's going on in the park at the same time."_

https://github.com/OpenRCT2/OpenRCT2/issues/7939

**Comment by ocalhoun6** [comment]
This is a good idea, but a bit complicated by the fact that it depends on screen resolution *and* zoom level.

**Comment by karst** [comment]
I wanted to suggest a different way of this. It'd be introducing a new option "Synchronize with ride" then have the drop down change from a list of ride music to a list of rides instead. Then these would all play the same audio track. 
The only thing to keep in mind is how breakdowns would then work. If the original ride breaks down will it copy the audio from that breakdown too and make all other places quiet, too? And what happens if a ride breaks down that copies? Will it also slow down the audio (or speed it up in the case of the merry-go-round?) or will it just stop emmiting audio till it's repaired? See #19711

Together with this I would like to see the introduction of sound emmiting scenery that can play ride music or synchronize with rides. Ofcourse this could never break down. Perhaps for the scenery it should always be synchronized to a ride. We could make it so the speakerboxes from Time Twister flower power would have this function, as well as some new speaker scenery objects, which can either be path tile elements or regular plopables, tiny (quater tile), small (one tile) and large (bigger than 1 tile).
